system-overview/cryptography.md

```diff
@@ -0,0 +1,115 @@
+keys. In both cases, these are public/private key pairs utilized for public key
+cryptography.
+
+#### Azimuth keys
+
+Your Urbit ID exists as an ERC-721 non-fungible token on the Ethereum
+blockchain, and as such is contained in a wallet whose private key you possess.
+If you are are utilizing a [master
+ticket](@/docs/azimuth/azimuth.md#master-ticket), this private key is derived
+from a seed, which is what you use to login to
+[Bridge](@/docs/glossary/bridge.md). Otherwise, you have generated the key by
+some other process, of which there are too many to list here. Besides the
+private key which unlocks your ownership wallet address, you may have a few
+other private keys which unlock a wallet that corresponds to your ship's
+[proxies](@/docs/glossary/proxies.md). We refer collectively to these keys as
+your _Azimuth keys_.
+
+Only [planets](@/docs/glossary/planet.md), [stars](@/docs/glossary/star.md), and
+[galaxies](@/docs/glossary/galaxy.md) have Azimuth keys.
+[Moons](@/docs/glossary/moon.md) and [comets](@/docs/glossary/comet.md) do not,
+as they do not exist on the Ethereum blockchain.
+
+It is important to note that no Azimuth keys are stored anywhere within your
+ship's [pier](@/docs/glossary/pier.md) - Ethereum and Urbit ID are entirely
+separate entities from Urbit itself, and so you lose access to your Azimuth
+private keys there is no way to retrieve them somehow from your ship.
+
+For more information on the usage of these keys and the associated proxies, see
+the [Azimuth documentation](@/docs/azimuth/azimuth.md).
+
+#### Networking keys
+
+All communications in Urbit over the [Ames](@/docs/glossary/ames.md) network
+are end-to-end encrypted, and thus your ship stores its own public/private pair
+of _networking keys_ utilized for encryption and authentication. Networking keys
+for all ship types are stored within the ship's [Jael](@/docs/glossary/jael.md)
+[vane](@/docs/glossary/vane.md).
+
+For planets, stars, and galaxies, your networking public key is configured on
+the Ethereum blockchain using one of your Azimuth private keys - either the one
+associated to the wallet which owns the ship, or the one which holds the
+[management proxy](@/docs/glossary/proxies.md). This is typically accomplished
+with Bridge. Your networking private key is a necessary input (the
+[keyfile](@/docs/glossary/keyfile.md)) for the initial boot sequence of your
+ship (sometime called its `%dawn`), and this is also provided by Bridge.
+
+Public keys for moons are not tracked on the blockchain; instead they are
+tracked by their parent ship. The networking private key is generated by the
+parent ship and injected into the `%dawn` sequence for the moon, and its public
+key is stored by the parent in its Jael vane. Networking keys for moons may be
+changed by the parent ship, but not by the moon itself.
+
+For comets, their 128-bit `@p` name is actually their networking public key, and
```

Comment:
The `@p` is actually the fingerprint, which is a _hash_ of the public key proper.